bin/ensuite-present.js

```diff
@@ -0,0 +1,23 @@
+#! /usr/bin/env node
+
+'use strict'
+
+const path = require('path')
+const http = require('http')
+const nodeStatic = require('node-static')
+
+const PORT = 4242
```

Comment:
I still think bespoke should propose a pseudo-random port but that's another disussion :smile: